test/e2e/performanceprofile/testdata/render-expected-output/manual_machineconfig.yaml

```diff
@@ -24,121 +24,121 @@ spec:
-            Environment=HUGEPAGES_SIZE=1048576
-            Environment=NUMA_NODE=0
-            Type=oneshot
-            RemainAfterExit=true
-            ExecStart=/usr/local/bin/hugepages-allocation.sh
+          [Service]
+          Environment=HUGEPAGES_COUNT=1
+          Environment=HUGEPAGES_SIZE=1048576
+          Environment=NUMA_NODE=0
+          Type=oneshot
+          RemainAfterExit=true
+          ExecStart=/usr/local/bin/hugepages-allocation.sh
 
-            [Install]
-            WantedBy=multi-user.target
-          enabled: true
-          name: hugepages-allocation-1048576kB-NUMA0.service
-        - contents: |
-            [Unit]
-            Description=Sets network devices RPS mask
+          [Install]
+          WantedBy=multi-user.target
+        enabled: true
+        name: hugepages-allocation-1048576kB-NUMA0.service
+      - contents: |
+          [Unit]
+          Description=Set cpus offline: 2,3
+          Before=kubelet.service
 
-            [Service]
-            Type=oneshot
-            ExecStart=/usr/local/bin/set-rps-mask.sh %i 00000001
-          name: update-rps@.service
-        - contents: |
-            [Unit]
-            Description=Set cpus offline: 2,3
-            Before=kubelet.service
+          [Service]
+          Environment=OFFLINE_CPUS=2,3
+          Type=oneshot
+          RemainAfterExit=true
+          ExecStart=/usr/local/bin/set-cpus-offline.sh
 
-            [Service]
-            Environment=OFFLINE_CPUS=2,3
-            Type=oneshot
-            RemainAfterExit=true
-            ExecStart=/usr/local/bin/set-cpus-offline.sh
+          [Install]
+          WantedBy=multi-user.target
+        enabled: true
+        name: set-cpus-offline.service
+      - contents: |
+          [Unit]
+          Description=Clear the IRQBalance Banned CPU mask early in the boot
+          Before=kubelet.service
+          Before=irqbalance.service
 
-            [Install]
-            WantedBy=multi-user.target
-          enabled: true
-          name: set-cpus-offline.service
-        - contents: |
-            [Unit]
-            Description=Clear the IRQBalance Banned CPU mask early in the boot
-            Before=kubelet.service
-            Before=irqbalance.service
+          [Service]
+          Type=oneshot
+          RemainAfterExit=true
+          ExecStart=/usr/local/bin/clear-irqbalance-banned-cpus.sh
 
-            [Service]
-            Type=oneshot
-            RemainAfterExit=true
-            ExecStart=/usr/local/bin/clear-irqbalance-banned-cpus.sh
-            
-            [Install]
-            WantedBy=multi-user.target
-          enabled: true
-          name: clear-irqbalance-banned-cpus.service
+          [Install]
+          WantedBy=multi-user.target
+        enabled: true
+        name: clear-irqbalance-banned-cpus.service
   extensions: null
   fips: false
   kernelArguments: null
   kernelType: realtime
-  osImageURL: ""
+  osImageURL: ""
```

Comment:
newline

Reply:
this passes the no cluster tests which generates and compare this file